composer.json

```diff
@@ -16,19 +16,20 @@
         }
     ],
     "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
         "php": ">=5.3.9",
-        "symfony/framework-bundle": "~2.3.27 || ~2.6.6 || ~2.7 || ~3.0",
+        "symfony/framework-bundle": "~2.3.27 || ^2.6.6 || ~3.0",
         "symfony/doctrine-bridge": "~2.3 || ~3.0",
-        "phpcr/phpcr-implementation": "2.1.*",
```

Comment:
this is tracking the version of jcr, which did not respect semver in the past. jackalope itself is semver, but this thing might not be.